woocommerce/woocommerce
# Removed unnecessary form change trigger on load of checkout page.

### All Submissions:






### Changes proposed in this Pull Request:



Closes #20640 . Please see the issue ticket for further analysis of what is being done on load and why I think it's not necessary.

Please test also using geolocation enabled checkout and predefined address checkout for different regions.

### How to test the changes in this Pull Request:

1. WC Settings > General > set  'Default customer location' to 'No location by default'.
2. Go to checkout page, check that paragraph with id `billing_country_field` has classes like `woocommerce-invalid` and  `woocommerce-invalid-required-field` asigned.
3. Apply the branch.
4. Check the checkout again, it should no longer assign those classes on load of the page. When the user selects a country and then reverts back to no country being selected, the classes are added.
5. WC Settings > General > set  'Default customer location' to 'Geolocate'.
6. Check that checkout page still displays forms correctly.

### Other information:




### Changelog entry

> Removed unnecessary checkout form change trigger on load of the checkout page.
